resources/building-mobile-extensions/updating-the-shared-state.md

```diff
@@ -2,24 +2,8 @@
 
 Extensions set a shared state by creating an event data that the extensions want to save and by calling `setSharedEventState` from the `ACPExtensionApi` \(iOS\) / `ExtensionApi` \(Android\) interface that is available through the `ACPExtension` \(iOS\) / `Extension` \(Android\) parent class. Extensions can only set their own state, so the name that is used is the same as your extension \(for example, _com.exampleCompany.extension_\). This name is also the name that other extensions need to use when requesting your shared state.
```

Comment:
The first sentence is confusing! Please change it to read as follows:

Extensions set a shared state by creating event data that the extensions save by calling `setSharedEventState` from the `ACPExtensionApi` \(iOS\) / `ExtensionApi` \(Android\) interface. 

is this part really necessary? --> "that is that is available through the `ACPExtension` \(iOS\) / `Extension` \(Android\) parent class." is necessary. 

If it is necessary, reword as follows "This interface is available through the `ACPExtension` \(iOS\) / `Extension` \(Android\) parent class."